packages/p2p/src/hooks/use-p2p-advertiser-adverts.ts

```diff
@@ -0,0 +1,41 @@
+import { useP2PAdvertList } from '@deriv/hooks';
+import { useStores } from 'Stores/index';
+
+/**
+ * @name useP2PAdvertiserAdverts
+ *
+ * @description This custom hook returns paginated available adverts for a given advertiser based on their advertiser_id by calling 'p2p_advert_list' endpoint
+ * @returns {object} The returned object contains many fields, some of which are explained below:
+ * object.loadMoreAdvertiserAdverts - A function to load more adverts.  (used by the InfiniteLoader @see https://github.com/bvaughn/react-virtualized/blob/master/docs/InfiniteLoader.md).
+ * object.has_more_adverts_to_load - Whether there are more adverts to load (used by the InfiniteLoader @see https://github.com/bvaughn/react-virtualized/blob/master/docs/InfiniteLoader.md).
+ * @example const { adverts, has_more_adverts_to_load, ...rest } = useP2PAdvertiserAdverts();
+ * */
+
+const useP2PAdvertiserAdverts = () => {
+    const { advertiser_page_store, general_store, buy_sell_store } = useStores();
+
+    const {
+        data: adverts = [],
+        loadMoreAdverts: loadMoreAdvertiserAdverts,
+        ...rest
+    } = useP2PAdvertList({
+        counterparty_type: advertiser_page_store.counterparty_type,
+        offset: 0,
```

Comment:
if we can modify the hook from my suggestions, we dont really need to pass this here, plus in your hook already, the default was already 0 if we don't pass anything, so no need to pass this here

```suggestion
```

Reply:
I'll remove this @ameerul-deriv, but I though being explicit would be better 🤔

Reply:
hmmm both ways are fine, but yes we can leave this in